test/print/memory-notshared.txt

```diff
@@ -1,3 +0,0 @@
-(module
```

Comment:
why is this test going away?

Reply:
because "notshared" isn't a keyword anymore; anything not "shared" is implicitly not shared.

Reply:
cool, lgtm